src/invoice.rs

```diff
@@ -90,10 +94,26 @@ pub enum InvoiceParseError {
     /// invalid invoice.
     Invalid,
 
+    #[display(doc_comments)]
+    /// invalid invoice: contract ID with but no iface
+    ContractIdNoIface,
+
     #[display(doc_comments)]
     /// invalid contract ID.
     InvalidContractId(String),
 
+    #[display(doc_comments)]
+    /// invalid interface
```

Comment:
This errors are printed to the end-used in the API (like in command-line - or I do that in MyCitadel as well). Thus we need more verbose explanation/providing details:
```suggestion
    /// invalid interface {0}.
```

Reply:
updated